### ISSUE TITLE
[Bug]: Unable to find installation candidates for primp (0.8.3)

### ISSUE THREAD

**Issue by southkorea2013** [infiniflow/ragflow#4082]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### RAGFlow workspace code commit ID

f8c9ec4  

### RAGFlow image version

v0.14.1-slim 

### Other environment information

```Markdown
OS: macOS 15.1 Apple silicon
```

### Actual behavior

docker build --build-arg ARCH=arm64 --build-arg LIGHTEN=1 --no-cache -f Dockerfile -t infiniflow/ragflow:v0.14.1-slim .

```
108.2   RuntimeError
108.2 
108.2   Unable to find installation candidates for primp (0.8.3)
108.2 
108.2   at ~/.local/pipx/venvs/poetry/lib/python3.10/site-packages/poetry/installation/chooser.py:74 in choose_for
108.2        70│ 
108.2        71│             links.append(link)
108.2        72│ 
108.2        73│         if not links:
108.2     →  74│             raise RuntimeError(f"Unable to find installation candidates for {package}")
108.2        75│ 
108.2        76│         # Get the best link
108.2        77│         chosen = max(links, key=lambda link: self._sort_key(package, link))
108.2        78│ 
108.2 
108.2 Cannot install primp.

```

### Steps to reproduce

```Markdown
docker build --build-arg ARCH=arm64 --build-arg LIGHTEN=1 --no-cache -f Dockerfile -t infiniflow/ragflow:v0.14.1-slim .
```

**Comment by jiyzhang** [comment]
same issue.
- macOS 15.2 Apple Silicon

**Comment by yuzhichang** [comment]
Fixed in v0.15.0
